src/lib.rs

```diff
@@ -1467,6 +1587,12 @@ impl CommandEncoder {
     }
 
     /// Copy data from a buffer to a texture.
+    ///
+    /// # Panics
+    ///
+    /// - Copy would overrun buffer.
+    /// - Copy would overrun texture.
+    /// - source.layout.rows_per_image isn't divisible by 256.
```

Comment:
Wouldn't this be `source.layout.bytes_per_row` rather than `source.layout.rows_per_image`?

Reply:
Yup! Messed this up at least once before...